WindowsServerDocs/failover-clustering/deploy-two-node-clustered-file-server.md

```diff
@@ -311,33 +311,52 @@ To configure a file server failover cluster, follow the below steps.
 
    ![File Server Type](media/Cluster-File-Server/Cluster-FS-File-Server-Type.png)
 
-8. In the **Client Access Point** window, input the name of the file server you will be using.  Please note that this is not the name of the cluster.  This is for the file share connectivity.  For example, if I want to connect to \\SERVER, the name inputted would be SERVER.
+8. In the **Client Access Point** window, input the name of the file server you are using.  Please note that this is not the name of the cluster.  
+This is for the file share connectivity.  
+For example, if you want to connect to  \\\\SERVER, specify the name as SERVER.
 
    > [!NOTE]
-   > If you are using static IP Addresses, you will need to select the network to use and input the IP Address it will use for the cluster name.  If you are using DHCP for your IP Addresses, the IP Address will be configured automatically for you.
+   > If you are using static IP addresses, you need to select the network to use and input the IP address to be used for the cluster name.  If you are using DHCP for your IP addresses, the IP address is configured automatically for you.
 
 6. Choose **Next**.
 
-7. In the **Select Storage** window, select the additional drive (not the witness) that will hold your shares and **Next**.
+7. In the **Select Storage** window, select the additional drive (not the witness) that holds your shares and **Next**.
 
 8. On the **Confirmation** page, verify your configuration and select **Next**.
 
-9. On the **Summary** page, it will give you the configuration it has created.  You can select View Report to see the report of the file server role creation.
+9. On the **Summary** page, it gives you the configuration it has created.  You can select View Report to see the report of the file server role creation.
 
-10. Under **Roles** in the console tree, you will see the new role you created listed as the name you created.  With it highlighted, under the **Actions** pane on the right, choose **Add a share**.
+10. Under **Roles** in the console tree, you see the new role you created listed as the name you created.  With it highlighted, under the **Actions** pane on the right, choose **Add a share**.
 
 11. Run through the share wizard inputting the following:
 
-    - Type of share it will be
-    - Location/path the folder shared will be
-    - The name of the share users will connect to
+    - Type of share it is
```

Comment:
Simplification (in contrast to expanding it to a natural sentence):
```suggestion
    - Type of share
```
Alternative: [The type of share it is]